packages/utils/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 9.1.1-beta.1
+
+-   feat(utils): scheduleAction with attempt failure handler (3e342aaa5b)
+
 # 9.0.25-beta.2
```

Comment:
the same here, previous beta but we did stable release, didn't we?

Reply:
yes, the previous release was stable.

Reply:
Right, last time we did for first time the release from beta to stable without any changes in between.
What we should do is the opposite, not generate changelog for connect dependencies when we are doing canary release and do it when we move from canary to stable. I am going to change the script now.

Reply:
that sounds about right